### PR TITLE
docs: use a custom bootstrap

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "bootstrap": "4.6.0",
```
Comment (jd) [inline package.json:3]: do we need to configure dependabot for this?

Comment (sileht) [reply package.json:3]: done

**Review by jd — COMMENTED** [review]
I don't think we need the `.nvimlog` file right?
should we move package.json  in the docs directory to be cleaner?

**Comment by sileht** [comment]
> I don't think we need the `.nvimlog` file right?

darn, it's a recent fu^W bug of my editor :(

> should we move package.json in the docs directory to be cleaner?

I can try

**Review by jd — APPROVED** [review]

**Pull request merged**